packages/headless/src/features/insight-search/insight-search-actions-loader.ts

```diff
@@ -0,0 +1,78 @@
+import {AsyncThunkAction} from '@reduxjs/toolkit';
+import {AsyncThunkInsightOptions} from '../../api/service/insight/insight-api-client';
+import {search} from '../../app/reducers';
+import {InsightEngine} from '../../insight.index';
+import {SearchAction} from '../analytics/analytics-utils';
+import {ExecuteSearchThunkReturn} from '../search/search-actions';
+import {
+  executeSearch,
+  fetchFacetValues,
+  fetchMoreResults,
+  StateNeededByExecuteSearch,
+} from './insight-search-actions';
+
+export interface InsightSearchActionCreators {
+  /**
+   * Creates an action that executes a search query.
+   *
+   * @example
+   *
+   * ```js
+   * const {logInterfaceLoad} = loadSearchAnalyticsActions(engine);
+   * const {executeSearch} = loadInsightSearchActions(engine);
+   *
+   * engine.dispatch(executeSearch(interfaceLoad()));
```

Comment:
I think you forgot to end the code block here.
```suggestion
   * engine.dispatch(executeSearch(interfaceLoad()));
   * ```
```

Reply:
good catch, thanks 🙏